### DISCUSSION TITLE
Working on AWS Sagemaker?

### DISCUSSION THREAD

**Discussion by p-somers** [TheLastBen/fast-stable-diffusion#562] [Ideas]
Just curious if anyone has gotten this working on the free tier of AWS sagemaker. I'm using up all my available resources on google colab lol

**Comment by TheLastBen** [comment]
I'll check it out

**Reply by p-somers** [reply]
Thanks!

**Reply by p-somers** [reply]
any luck?

**Reply by TheLastBen** [reply]
not yet, maintaining these colab notebooks take most of my free time.

**Reply by nawnie** [reply]
I may look into it i've signed up but havn't tried it out yet. just a suggestion, but if you where to say make a model to post on reddit tag your email in said post and say your accepting request whether you do the request or not that would give you a valid second google account you could claim is used for public contact to keep you personal gmail separate or just make a second gmail :D

**Reply by i-huzaifa-arshad** [reply]
It would be a huge plus. Sagemaker Studio Lab offers free 12-hour CPU and 4-hour GPU access. If you can manage to create notebooks (A1111 Webui and Dreambooth ones) for it, it would be amazing.

**Comment by Algoworks11** [comment]
Can we use fast stable diffusion on AWS?
